9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init ]; then
```

Comment:
We could also check that the file is executable with `-x`?

```bash
[ -f "${CONTAINER_SCRIPTS_PATH}/post-init" -a -x "${CONTAINER_SCRIPTS_PATH}/post-init" ] 
```

Maybe there's a better way to write the above.

If the file exists and is not executable, with just the `-f` test one could get an error like:

```
-bash: /opt/app-root/scripts/post-init: Permission denied
```

And it may not be very obvious what needs to be done to fix it (from the user's perspective).